apps/web/screens/queries/Frontpage.ts

```diff
@@ -0,0 +1,52 @@
+import gql from 'graphql-tag'
+
+export const GET_FRONTPAGE_QUERY = gql`
+  query GetFrontpage($input: GetFrontpageInput!) {
+    getFrontpage(input: $input) {
```

Comment:
Maybe we should add `id` field here to better support cache:
https://www.apollographql.com/docs/react/caching/cache-configuration/#default-identifier-generation